### ISSUE TITLE
Some UTF8 in Fragments can't be parsed / fixed

### ISSUE THREAD

**Issue by stroomdev10** [gchq/stroom#3057]
Run the attached stream through the attached pipeline to get the error below.

I think 0x8n are part of a multibyte, but something is getting confused

```[1:1:5852 xmlFragmentParser] FATAL: An invalid XML character (Unicode: 0x80) was found in the element content of the document.
[xmlFragmentParser] FATAL: An invalid XML character (Unicode: 0x80) was found in the element content of the document.
[invalidXMLCharFilterReader] WARN: The content was modified
[PipelineStreamProcessor] FATAL: An invalid XML character (Unicode: 0x80) was found in the element content of the document.```

**Comment by stroomdev10** [comment]
[Samples.zip](https://github.com/gchq/stroom/files/9574051/Samples.zip)
[StroomData.zip](https://github.com/gchq/stroom/files/9574053/StroomData.zip)

**Comment by at055612** [comment]
Fixed in v7.0.2
